### PR TITLE
Chapter 13 cleanup

### PR DESCRIPTION
**Pull request by bast**
Progress:
- [x] CI
- [x] 13/1 apply reviewer's comments
- [x] 13/2 apply reviewer's comments
- [x] 13/3 apply reviewer's comments
- [x] 13/1 cleanup and TC in sync
- [x] 13/2 cleanup and TC in sync
- [x] 13/3 cleanup and TC in sync
- [x] place follow-up actions (if any) to TODO document
- [x] ready to go

### CODE REVIEW

File: chapter-13/recipe-01/cxx-example/CMakeLists.txt [hunk 1]
```diff
@@ -20,7 +20,7 @@ add_executable(hello-world hello-world.cpp)
 target_compile_definitions(
   hello-world
   PUBLIC
-    "-DCMAKE_SYSTEM_NAME=\"${CMAKE_SYSTEM_NAME}\""
+    "-DSYSTEM_NAME=\"${CMAKE_SYSTEM_NAME}\""
```
Comment (robertodr) [inline chapter-13/recipe-01/cxx-example/CMakeLists.txt]: `-D` is not needed (and we haven't used elsewhere) This suffices:
```
target_compile_definitions(hello-world PUBLIC "COMPILER_NAME=\"${CMAKE_CXX_COMPILER_ID}\"")
```

Comment (robertodr) [reply chapter-13/recipe-01/cxx-example/CMakeLists.txt]: BTW, I've done this in TC already.

**Comment by bast** [comment]
Thanks! Yes, I accidentally removed Appveyor testing when cleaning up my commit history and this morning when I woke up I remembered this but good that you caught it, too.

**Comment by bast** [comment]
Waiting for final round of CI and will merge then but do not expect any failures.

**Pull request merged**